internal/util/path.go

```diff
@@ -18,8 +18,23 @@ func GetApplicationDir() (string, error) {
 		return "", err
 	}
 
+	legacyFolder := false
+
+	// check if the home/.twitch-cli folder exists; if so, use that as the path
+	if _, err := os.Stat(filepath.Join(home, ".twitch-cli")); !os.IsNotExist(err) {
+		legacyFolder = true
+	}
+
 	path := filepath.Join(home, subFolder)
 
+	if !legacyFolder {
+		path = filepath.Join(home, ".config", subFolder) // issue #33- putting into a subfolder to avoid clutter
```

Comment:
Given subFolder has the dot prefix, this would put the config in `$HOME/.config/.twitch-cli/` right? I don't believe this is expected when it is in the `.config` directory already.

Reply:
Correct; is the expectation that it'd omit the dot prefix once in there?

Reply:
I assume that not having them in .config is the norm yes. Checking my .config directory, I didn't see a single dot prefix out of more than 100.

Reply:
fixed; resolving this.